### PR TITLE
added solution for homework Builder

### PR DESCRIPTION
**Pull request by Valdes2**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,4 +7,96 @@
...
+        private String modelName;
+        private int emptyWeight;
+        private int takeoffWeight;
+        private int cruisingSpeed;
+        private int enginePower;
+        private int wingspan;
+
+        public PlaneBuilder setModelName(String modelName) {
+            this.modelName = modelName;
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: empt line not required

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,4 +7,96 @@
...
+            return this;
+        }
+
+        public PlaneBuilder setEmptyWeight(int emptyWeight) {
+            this.emptyWeight = emptyWeight;
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: empt line not required

File: src/main/java/core/basesyntax/Plane.java [hunk 3]
```diff
@@ -7,4 +7,96 @@
...
+        public PlaneBuilder setEnginePower(int enginePower) {
+            this.enginePower = enginePower;
+
+            return this;
+        }
+
+        public PlaneBuilder setWingspan(int wingspan) {
+            this.wingspan = wingspan;
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: empt line not required

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Plane.java (hunk 3 above)
Comment (Valdes2) [reply src/main/java/core/basesyntax/Plane.java]: all fixed

**Pull request closed without merging**